devtoolsNG/xen/tools/gister.html

```diff
@@ -0,0 +1,4 @@
+<script type="module">
```

Comment:
Add copyright?

Reply:
Yes, I suppose. This module will then be 60% non-functioning dross, which I find irritating.

Reply:
My annoyance aimed at lawyercats not you, friendly reviewer. :)